plugins/woocommerce-beta-tester/api/options/rest-api.php

```diff
@@ -40,6 +41,42 @@
 	)
 );
 
+register_woocommerce_admin_test_helper_rest_route(
+	'/transients',
+	'wca_test_helper_get_transient',
+	array(
+		'methods' => \WP_REST_Server::READABLE,
+		'args'                => array(
+			'search'   => array(
+				'description'       => 'Limit results to those matching a string.',
+				'type'              => 'string',
+				'sanitize_callback' => 'sanitize_text_field',
+			),
+		),
+	)
+);
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/transients',
+	'wca_test_helper_add_transient',
+	array(
+		'methods' => \WP_REST_Server::EDITABLE,
+	)
+);
+
+register_woocommerce_admin_test_helper_rest_route(
+	'/transients/(?P<transient_names>(.*)+)',
+	'wca_test_helper_delete_option',
```

Comment:
This is still referring to the `delete_option` callback, I assume we should create a `transient` specific callback for this?
It might be worth still calling the `delete_option` as well, for any old experiment data.